### PR TITLE
gpu: regions: add various areas

### PR DESCRIPTION
**Pull request by cdfisher**
Adds GPU regions for the following:

- HAM Storeroom
- Mor Ul Rek/Fight Pit
- ~~Fight Cave~~
- Barbarian Assault lobby
- Mage Training Arena
- Water Ravine Dungeon
- Kalphite Cave
- Kalphite Lair
- Chasm of Tears

### CODE REVIEW

File: runelite-client/src/main/resources/net/runelite/client/plugins/gpu/regions/regions.txt [hunk 1]
```diff
@@ -307,4 +307,51 @@ R 36 147 37 147
...
+
+// mta telekenetic theatre
+n
+r 52 151
+
+// water ravine (spirits of the elid)
+n
+r 52 149
+m 52 148
+c 2 7
```
[the remark this answers is not in the document]
Comment (cdfisher) [reply runelite-client/src/main/resources/net/runelite/client/plugins/gpu/regions/regions.txt:341]: It's visible from the Dorgeshuun Mines but seemed to be simpler to map out as a region than doing all of the connected Lumbridge caves would have been.
water ravine

File: runelite-client/src/main/resources/net/runelite/client/plugins/gpu/regions/regions.txt [hunk 2]
```diff
@@ -307,4 +307,51 @@ R 36 147 37 147
...
+n
+r 51 148
+m 51 149
+C 0 0 7 1
+m 52 148
+C 0 0 2 6
+
+// kalphite lair
+n
+r 54 148
```
[the remark this answers is not in the document]
Comment (cdfisher) [reply runelite-client/src/main/resources/net/runelite/client/plugins/gpu/regions/regions.txt:353]: kalphite lair
kalphite lair after

File: runelite-client/src/main/resources/net/runelite/client/plugins/gpu/regions/regions.txt [hunk 3]
```diff
@@ -307,4 +307,51 @@ R 36 147 37 147
...
+r 40 81
+
+// mor ul rek
+n
+r 37 80
+R 38 79 39 80
+
+// fight caves
+n
+r 37 79
```
[the remark this answers is not in the document]
Comment (cdfisher) [reply runelite-client/src/main/resources/net/runelite/client/plugins/gpu/regions/regions.txt:323]: This one does look to be unnecessary, even with high draw distance and extended map loading.

[the remark this answers is not in the document]
Comment (Adam-) [reply runelite-client/src/main/resources/net/runelite/client/plugins/gpu/regions/regions.txt:323]: The fight caves is in an instance, so I don't think it is possible to see anything else.

**Pull request merged**